system/cc-ceph/templates/prysm-rgw-sidecar-webhook-cert.yaml

```
@@ -0,0 +1,14 @@
+{{- if and .Values.objectstore.prysm.rgwSidecarWebhook ( .Values.objectstore.prysm.rgwSidecarWebhook.enabled ) }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ .Values.objectstore.prysm.rgwSidecarWebhook.certSecretName }}
+  namespace: {{ .Values.objectstore.prysm.rgwSidecarWebhook.namespace }}
```

```suggestion
  namespace: {{ .Release.Namespace }}
```

and in other places too